ui/pages/swaps/build-quote/build-quote.js

```diff
@@ -392,6 +410,51 @@ export default function BuildQuote({
     fromTokenAddress,
     selectedToToken.address,
   );
+  const isReviewSwapButtonDisabled =
+    tokenFromError ||
+    !isFeatureFlagLoaded ||
+    !Number(inputValue) ||
+    !selectedToToken?.address ||
+    Number(maxSlippage) < 0 ||
+    Number(maxSlippage) > MAX_ALLOWED_SLIPPAGE ||
+    (toTokenIsNotDefault && occurrences < 2 && !verificationClicked);
```

Comment:
Not your fault, but this conditional is massive and maybe a tiny bit confusing.  I don't have a solution, but maybe we should think on this in the future.

Reply:
Agree with you! I've just moved all these conditions into a constant, but they were there previously. I will keep an eye on this and will refactor it / add explanatory comments if it would be getting any more complex.